apps/language_server/lib/language_server/build.ex

```diff
@@ -10,7 +10,15 @@ defmodule ElixirLS.LanguageServer.Build do
         with_build_lock(fn ->
           {us, _} =
             :timer.tc(fn ->
-              IO.puts("Compiling with Mix env #{Mix.env()}")
+              IO.puts("MIX_ENV: #{Mix.env()}")
+
+              if function_exported?(Mix, :target, 0) do
+                # Even though we know we have the function here,
+                # Compilation will fail calling Mix.target/0 directly
+                # since Elixir 1.7 is used to compile - So we get around
```

Comment:
Maybe it's time we dropped support for 1.7

Reply:
This was brought up [in the original issue](https://github.com/elixir-lsp/elixir-ls/issues/248#issuecomment-633146407), but ultimately decided that things weren't ready to drop the minimum to away from 1.8 - But I'm definitely open to suggestions about making this workaround look better if desired?